presto-spi/src/test/java/com/facebook/presto/spi/block/TestArrayBlockBuilder.java

```diff
@@ -48,6 +49,20 @@ private void testIsFull(PageBuilderStatus pageBuilderStatus)
         assertEquals(pageBuilderStatus.isFull(), true);
     }
 
+    @Test
+    public void testRetainedSizeInBytes()
+    {
+        final int expectedEntries = 1000;
```

Comment:
Add a TODO that we should a systematically way to test `getRetainedSizeInBytes` on all Blocks.